src/NHibernate/Dialect/SybaseSQLAnywhere17Dialect.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System.Data;
+using System.Data.Common;
+using NHibernate.Dialect.Schema;
+using Environment = NHibernate.Cfg.Environment;
+
+namespace NHibernate.Dialect
+{
+	/// <remarks>
+	/// The SybaseSQLAnywhere17Dialect uses the SybaseSQLAnywhere12Dialect as its 
+	/// base class. SybaseSQLAnywhere17Dialect includes support for ISO SQL standard
+	/// sequences, which are defined in the catalog table <tt>SYSSEQUENCE</tt>. 
+	/// The dialect uses the SybaseSQLAnywhe12MetaData class for metadata API
+	/// calls, which correctly supports reserved words defined by SQL Anywhere.
+	/// 
+	/// The dialect defaults the following configuration properties:
+	/// <list type="table">
+	///	<listheader>
+	///		<term>Property</term>
+	///		<description>Default Value</description>
+	///	</listheader>
+	///	<item>
+	///		<term>connection.driver_class</term>
+	///		<description><see cref="NHibernate.Driver.SybaseSQLAnywhere17Driver" /></description>
+	///	</item>
+	///	<item>
+	///		<term>prepare_sql</term>
+	///		<description><see langword="false" /></description>
+	///	</item>
+	/// </list>
+	/// </remarks>
+	public class SybaseSQLAnywhere17Dialect : SybaseSQLAnywhere12Dialect
```

Comment:
It seems to me the name Sybase is phased out. Shouldn't it be named `SapSQLAnywhere17Dialect`?

Reply:
Yes, that's correct! It should be renamed to Sap instead.